test/clj/rems/api/test_api.clj

```diff
@@ -80,7 +83,7 @@
         (testing path
           (is (response-is-ok? (api-response :get path nil
                                              "46" "owner")))))
-      (is (response-is-unauthorized? (api-response :get "/api/catalogue" nil
+      (is (response-is-forbidden? (api-response :get "/api/catalogue" nil
```

Comment:
you could use `/api/applications/todo` in this case

Reply:
/api/applications/todo not sure what that would do?

Reply:
This test is just testing the api-key whitelisting logic, it doesn't really matter which (valid) APIs we use here. Now that /api/catalogue becomes public, it's not a good test target.